WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModel.cs

```diff
@@ -82,48 +75,76 @@ public void OpenWallet(bool receiveDominant)
 				Actions.Add(sendTab);
 			}
 
-			var receiveTab = new ReceiveTabViewModel(Wallet);
-			var coinjoinTab = new CoinJoinTabViewModel(Wallet);
-			var historyTab = new HistoryTabViewModel(Wallet);
+			_receiveTab = new ReceiveTabViewModel(Wallet);
```

Comment:
Please use a private property instead of fields to be consistent with the rest of the code.

Reply:
done.